bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/RetryingReadRowsOperation.java

```diff
@@ -203,6 +204,25 @@ public void onMessage(ReadRowsResponse message) {
     }
   }
 
+  @Override
+  /**
+   * Override the default behavior of {@link AbstractRetryingOperation#getRpcCallOptions()} to ensure that deadlines
+   * are not set for streaming reads.
+   */
+  protected CallOptions getRpcCallOptions() {
```

Comment:
I failed to see why this is needed? The parent class already know whether this is steaming call or not, no? Has to do with 'request' vs 'nextRequest'?

Reply:
I'll remove it.